src/components/Carousel/Carousel.js

```diff
@@ -52,15 +53,15 @@ const MyCarousel = () => {
   );
 }
 
-function RenderSlides({posts}) {
-  let slides = [];
-  for (let i = 0; i < posts.length; i++) {
-    slides.push(<Slide key={i} index={i}><SlideLocation info={posts[i]}></SlideLocation></Slide>);
-  }
+function RenderSlides({beaches}) {
   return (
     <Slider className={styles.slider}>
-          {slides}
-        </Slider>
+      {
+        beaches.map((beach, index) => 
+          <Slide key={beach.id} index={index}><SlideLocation info={beach}></SlideLocation></Slide>
```

Comment:
make sure to set a unique key that references the data, and never use a array index as a key. you might as well not put a key if you do that